src/sass/pages/index.scss

```diff
@@ -170,48 +198,54 @@
 
 .roadmap {
   @include corporate-gradient;
-  padding: $padding-roadmap-vertical 0;
   overflow: hidden;
+  color: $color-roadmap-text;
 
-  & article,
   & > div {
     @include row;
     @include container;
     position: relative;
+    padding: $padding-section-vertical 0;
   }
 
   .code-pills-left {
     position: absolute;
-    right: 95%;
+    right: 100.7%;
```

Comment:
Yes, they're numbers that make the code pills be exactly aligned in a certain position